libs/api/domains/vehicles/src/lib/api-domains-vehicles.service.ts

```diff
@@ -162,6 +190,11 @@ export class VehiclesService {
       return null
     }
 
+    const hasAuth = this.hasVehicleServiceAuth(auth, input.permno)
+    if (!hasAuth) {
+      return null
+    }
```

Comment:
This feels iffy. Returning null here can either mean no cars found (i think) or the user is unauthorized which are are 2 very different things.

Reply:
Yeah this should probably not return anything at all. A call to the auth service would be enough because it will throw an error on no auth anyway.